include/alpaka/rand/RandStl.hpp

```diff
@@ -43,6 +43,131 @@ namespace alpaka
             using RandBase = RandStl;
         };
 
+        namespace generator
+        {
+            namespace cpu
+            {
+                //#############################################################################
+                //! The STL mersenne twister random number generator.
+                //#############################################################################
+                class MersenneTwister
+                {
+                public:
+
+                    //-----------------------------------------------------------------------------
+                    //! Constructor.
+                    //-----------------------------------------------------------------------------
+                    ALPAKA_FN_ACC_NO_CUDA MersenneTwister() = default;
```

Comment:
I saw it currently in my tests: `ALPAKA_FN_ACC_NO_CUDA` must be removed to avoid the warning

```
alpaka/include/alpaka/rand/RandStl.hpp(60): warning: __host__ annotation on a defaulted function("MersenneTwister") is ignored

```

Reply:
I will fix it later